travis/build-samples.sh

```diff
@@ -20,8 +20,8 @@ function buildProject() {
     dir=$1
     echo "*********************** Building $dir"
     pushd "$mydir/samples/$dir"
-    npm link generator-jhipster
     yo jhipster --force
+    npm link generator-jhipster
```

Comment:
Why? Linking is quite useful because we can jump into the sample and try things like entity generation right after having built it.